src/Service/Startup.cs

```diff
@@ -355,6 +357,16 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env, RuntimeC
                 app.UseHttpsRedirection();
             }
 
+            // SwaggerUI visualization of the OpenAPI description document is only available
+            // in developer mode in alignment with the restriction placed on ChilliCream's BananaCakePop IDE.
+            if (runtimeConfigProvider.IsDeveloperMode() || env.IsDevelopment())
+            {
+                app.UseSwaggerUI(c =>
+                {
+                    c.SwaggerEndpoint("/openapi", "DataApibuilder-OpenAPI-PREVIEW");
```

Comment:
why `PREVIEW` ?

Reply:
I want to be explicitly clear that this feature is preview and the feature will have future updates/iterations .